examples/billing/add_funds.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+credit_card = EasyPost::CreditCard.fund('2000', 'primary')
```

Comment:
FWIW, This function will need to be updated once the Billing functions PR is merged, same for `CreditCard.delete()` and `PaymentMethod.all`

Reply:
This PR is merged, you will need to update the code example. You can find the unit tests for Billing function at here: https://github.com/EasyPost/easypost-ruby/blob/master/spec/billing_spec.rb